config/newrelic.yml

```diff
@@ -0,0 +1,49 @@
+#
+# This file configures the New Relic Agent.  New Relic monitors Ruby, Java,
+# .NET, PHP, Python and Node applications with deep visibility and low
+# overhead.  For more information, visit www.newrelic.com.
+#
+# Generated December 05, 2015
+# 
+# This configuration file is custom generated for app30169872@heroku.com
+#
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: 37035e8b605929eec1e27b4958b89248af195522
```

Comment:
Yikes! You're 100% right. Not that this API key grants much power but I'll do so